vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -400,6 +403,71 @@ public void twoStudiesConflictingNamesSingleVariant() {
         checkVariantContext(variantContext, CHR_1, 1000, 1000, "T", "G", variant.getSourceEntries(), true);
     }
 
+
+    @Test
+    public void csqAnnotation() {
```

Comment:
It would be good to test the creation with different sets of fields to ensure that all branches are evaluated. For instance, one method testing CSQ when there are/aren't SO terms, gene IDs, etc.